src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,76 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+import { useState } from 'react';
+import { Status } from '../../types/Status';
+
+type TodoFilterProps = {
+  onFilter: (status: string) => void,
```

Comment:
```suggestion
  onFilter: (status: Status) => void,
```

Reply:
fix it everywhere
<img width="949" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-todos/assets/70220667/9bd9ffaf-f08e-47a3-af30-1ad7d708de14">